src/svm/web3-v1/solanaProgramUtils.ts

```diff
@@ -239,6 +239,8 @@ export function stringifyCpiEvent(obj: any): any {
     return obj.toString();
   } else if (BN.isBN(obj)) {
     return obj.toString();
+  } else if (typeof obj === "bigint" && obj !== 0n) {
```

Comment:
why `0n` is excluded?

Reply:
Big bad, i put that during some tests and forgot about it. 
Thanks for catchinng!